### ISSUE TITLE
File Search Feature request.

### ISSUE THREAD

**Issue by fenlasonb** [RetroShare/RetroShare#1434]
Was just wondering if there is any chance there would be able to be a patch maybe or next version to be able to search directory names when using the search feature in Files. I think this would really help as my friends and I are using this as a mesh network sharing solution but we cannot search file directories which in case all would have a directory called movies and then in movies we can then open directories from there.

Just an idea for a nice feature request.

:)

cheers

**Comment by felisucoibi** [comment]
agree, in search, also include FOLDERS of friends file lists if match.  Is something we asked a few years ago and still waiting.

Meanwhile, in coming 0.6.5 you can search in FILELISTS, and it match folders i think. You can try the latest beta.

**Comment by fenlasonb** [comment]
Thanks for the update.

Is there a link for the beta?

Wish i could bring value to this. sounds like its been a feature request for sometime now :/

**Comment by felisucoibi** [comment]
If you are inside rs network there are channels with betas, which version are you uisng now?

**Comment by fenlasonb** [comment]
RetroShare Version: 0.6.4 Revision 91634ba6
Windows 10 (10.0) QT 5.9.1

libretroshare
 - bzip2: 1.0.6, 6-Sept-2010
 - OpenSSL: OpenSSL 1.0.2n  7 Dec 2017
 - SQLite: 3.7.17
 - SQLCipher: 2.2.1
 - UPnP (MiniUPnP): 2.0
 - Zlib: 1.2.3

RetroShare
 - Libmicrohttpd: 0.9.59
